cloudmapper/gatherer.py

```diff
@@ -11,24 +12,35 @@ def datetime_handler(x):
 
 
 def gather(arguments):
+    account_dir = './{}'.format(arguments.account_name)
+    
+    if arguments.clean and path.exists(account_dir):
+        rmtree(account_dir)
+
     try:
-        mkdir('./{}'.format(arguments.account_name))
+        mkdir(account_dir)
     except OSError:
-        # Already exists
+        # Already exists 
         pass
 
     print("* Getting region names")
-    ec2 = boto3.client('ec2')
+    session_data = {}
+
+    if arguments.profile_name:
+        session_data['profile_name'] = arguments.profile_name
+
+    session = boto3.Session(**session_data)
+    ec2 = session.client('ec2')
 
     region_list = ec2.describe_regions()
-    with open("./{}/describe-regions.json".format(arguments.account_name), 'w+') as f:
```

Comment:
I guess the reasoning behind removing the `./` is that it's just not needed?  Makes sense.

Reply:
so, I did not remove the "./" but moved it to the `account_dir` (https://github.com/alanquillin/cloudmapper/blob/904c08861735062e91518aba644f64b8d83c291d/cloudmapper/gatherer.py#L15).